### DISCUSSION TITLE
Urgent: Account Suspension Issue

### DISCUSSION THREAD

**Discussion by FreedomForSe7en** [community/community#130541] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this message finds you well. I am writing to seek urgent assistance regarding an issue I am experiencing with my GitHub account. My account has been suspended, and I am unable to push my repositories from Visual Studio Code to GitHub. This issue is significantly impacting my workflow and project timelines.

Account Details:

Username: FreedomForSeven
Email Address: freedomforseven@gmail.com

Issue Description: When attempting to push my code to GitHub from Visual Studio Code, I receive the following error message: “Git: remote: Your account is suspended. Please visit https://support.github.com for more information.”

I have recently created this account and have ensured that I am not violating any of GitHub’s terms of service. Despite this, my account remains suspended, and I am unable to proceed with my work. I have also tried the following troubleshooting steps without success:

Cleared my Git credentials from the system’s credential manager.
Reconfigured my Git settings in Visual Studio Code.
Attempted to use an alternate GitHub account, but the issue persists.
Impact: This suspension is causing significant delays in my project, as I am unable to push updates and collaborate with my team. It is crucial for me to have access to my GitHub account to continue my work and meet project deadlines.

Request: I kindly request your assistance in resolving this issue as soon as possible. Please provide any information regarding the reason for the suspension and the steps I need to take to restore my account. If there are any specific actions required from my end, please let me know, and I will address them promptly.

Thank you for your attention to this matter. I appreciate your prompt assistance and look forward to resolving this issue swiftly.

Best regards,
![Screenshot 2024-07-01 042217](https://github.com/community/community/assets/174294245/abc2add8-bcba-44b1-857a-87f67acd12ee)

**Comment by LiteBrite82** [comment]
Hi @FreedomForSe7en 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
